pyseir/ensembles/ensemble_runner.py

```diff
@@ -158,19 +207,18 @@ def init_run_mode(self):
 
             if len(self.covid_data) > 0 and self.covid_data.cases.max() > 0:
                 self.t0 = self.covid_data.date.max()
+                self.t0, hospitalizations_total = self.get_initial_hospitalizations()
 
-                # Initial hospitalizations = Confirmed Cases / 4.
-                # TODO Integrate actual hospital data here.
-                hospitalization_to_confirmed_case_ratio = 1 / 4
-
-                self.override_params['HGen_initial'] = self.covid_data.cases.max() * hospitalization_to_confirmed_case_ratio * (1 - self.override_params['hospitalization_rate_icu'])
-                self.override_params['HICU_initial'] = self.covid_data.cases.max() * hospitalization_to_confirmed_case_ratio * self.override_params['hospitalization_rate_icu']
+                self.override_params['HGen_initial'] = hospitalizations_total * (1 - self.override_params['hospitalization_rate_icu'])
+                self.override_params['HICU_initial'] = hospitalizations_total * self.override_params['hospitalization_rate_icu']
                 self.override_params['HICUVent_initial'] = self.override_params['HICU_initial'] * self.override_params['fraction_icu_requiring_ventilator']
+                self.override_params['I_initial'] = hospitalizations_total / self.override_params['hospitalization_rate_general']
```

Comment:
It is most correct to have a multiplicative delay factor here, roughly equal to: 

[serial_period]/[observed_doubling_time]

we may not need it for the first version here?